app/components/button_link_to/view.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ButtonLinkTo
+  class View < GovukComponent::Base
+    attr_reader :body, :url, :class_option, :html
+
+    def initialize(body:, url:, class_option: nil, **html)
+      @body = body
+      @url = url
+      @class_option = class_option
+      @html = html
+    end
+
+    def call
+      html_options = {
+        role: "button",
+        data: { module: "govuk-button" },
+        draggable: false,
+      }.merge({ class: class_option }, html)
+
+      html_options[:class] = prepend_css_class("govuk-button", html_options[:class])
```

Comment:
Out of interest, does the `govuk-button` class need to be in front of the others?

I think we could replace this line with `html_options[:class] += "govuk-button" if html_options["class]` then we dont need this `prepend_css_class` method. What do you think?

Reply:
TIL https://css-tricks.com/the-order-of-css-classes-in-html-doesnt-matter/

Reply:
I don't think the order mattes.